### ISSUE TITLE
Loading package with build number in version field

### ISSUE THREAD

**Issue by christopherjbaker** [stealjs/steal#1201]
__How often can you reproduce it?__ 

- [X] Always
- [ ] Sometimes
- [ ] Rarely
- [ ] Unable
- [ ] I didn’t try

__Description:__

I cannot load a module which has a build number in its version number, such as `4.0.0-alpha.6`.

__Steps to reproduce:__

1. Add a dependency to `bootstrap@4.0.0-alpha.6`.
2. Try to import `bootstrap`. See error of "Did not find ./node_modules/bootstrap/package.json".
3. Edit mentioned file to be version `4.0.0` (removing the build number). See that it loads properly.

__Expected results:__

Loads file.

__Actual results:__

Errors with "Did not find ./node_modules/bootstrap/package.json"

__Environment:__

| Software                     | Version
| --------------------- | -------
| Steal version              |  1.5.2
| Steal-tools version    |  1.3.3
| node -v                       |  6.10.2
| npm -v                        |  3.10.10
| yarn -v                        | 0.24.4
| Browser                      |  Chrome 58.0.3029.110 (64-bit)
| Operating system      |  MacOS 10.12.5

**Comment by matthewp** [comment]
I don't understand what you mean by "build number"? What does your dep look like in package.json?

**Comment by chasenlehara** [comment]
How are you importing Bootstrap? I’ve successfully used `can-import` with `bootstrap@4.0.0-alpha.6` in a DoneJS app.

**Comment by christopherjbaker** [comment]
Package.json contains
```json
  "dependencies": {
    "bootstrap": "4.0.0-alpha.6"
  }
```

and the javascript file contains
```javascript
import 'bootstrap/dist/css/bootstrap.css';
```

**Comment by matthewp** [comment]
I created a gist with this version of bootstrap working. https://gist.github.com/matthewp/3179d4c2c0abc6fce89948f45f07b21f

Make sure you have `"steal": { "plugins": [ "steal-css" ] }` in your package.json

**Comment by christopherjbaker** [comment]
TLDR: package.json said "4" so installed package of "4.0.0-alpha.6" did not match (since steal does version checks), but changing installed to show "4.0.0" matched.

The project that wasn't working is now working, but the fix I had to make to get it to work is not in place anymore. I think I know what the problem is, however. I was using yarn, and had run `yarn add bootstrap@4`, which then presented me a list of bootstrap versions, and I choose "4.0.0-alpha.6". This left my package.json reading `"bootstrap": "4"` however, and each time i ran `yarn add ...` it gave the same prompt. I fixed the package.json to prevent the prompts some time after opening the issue but before answering about my dependencies. Having both reflect "4.0.0-alpha.6" fixes the issue.
